drone_circuit_assets/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(jderobot_assets)
```

Comment:
I would suggest to change package name. `jderobot_assets` is an old and deprecated dependency